libfaasmp/faasmp.imports

```diff
@@ -32,9 +32,3 @@ omp_get_max_active_levels
 omp_set_default_device
 omp_set_max_active_levels
 
-# Redis state
-__faasmp_incrby
-__faasmp_getLong
-
-# Debug functions
-__faasmp_debug_copy
```

Comment:
This was some custom work done around using Redis for reduction. We'll be changing it so I removed it.